pkg/api-server/inspect_endpoints.go

```diff
@@ -547,3 +555,67 @@ func gatewayEntriesByPolicy(
 
 	return policyMap
 }
+
+func inspectRulesAttachment(cfg *kuma_cp.Config, builder xds_context.MeshContextBuilder) restful.RouteFunction {
+	return func(request *restful.Request, response *restful.Response) {
+		ctx := request.Request.Context()
+		meshName := request.PathParameter("mesh")
+		dataplaneName := request.PathParameter("dataplane")
+
+		meshContext, err := builder.Build(ctx, meshName)
+		if err != nil {
+			rest_errors.HandleError(response, err, "Could not build MeshContext")
+			return
+		}
+
+		matchedPolicies, _, proxy, err := getMatchedPolicies(
+			request.Request.Context(), cfg, meshContext, core_model.ResourceKey{Mesh: meshName, Name: dataplaneName},
+		)
+		if err != nil {
+			rest_errors.HandleError(response, err, "Could not get MatchedPolicies")
+			return
+		}
+		rulesAttachments := core_xds.BuildRulesAttachments(matchedPolicies.Dynamic, proxy.Dataplane.Spec.Networking)
+		sort.SliceStable(rulesAttachments, func(i, j int) bool {
+			if rulesAttachments[i].Name == rulesAttachments[j].Name {
+				return rulesAttachments[i].Type < rulesAttachments[j].Type
+			}
+			return rulesAttachments[i].Name < rulesAttachments[j].Name
+		})
```

Comment:
Should this sort go in `BuildRulesAttachements()`?

Reply:
Yeah, I wasn't sure, but maybe it should in the end since it's used only there anyways.